test/NuGet.Core.Tests/NuGet.ProjectModel.Test/DependencyGraphSpecTests.cs

```diff
@@ -731,7 +768,7 @@ private static DependencyGraphSpec CreateDependencyGraphSpecWithCentralDependenc
             packageSpec.RestoreMetadata = new ProjectRestoreMetadata() { ProjectUniqueName = "a", CentralPackageVersionsEnabled = true };
             var dgSpec = new DependencyGraphSpec();
             dgSpec.AddRestore("a");
-            dgSpec.AddProject(packageSpec);
+            dgSpec.AddProject(packageSpec.ApplyCentralVersionInformation());
```

Comment:
I think ideally we want to have this run pretty much in every test. Don't have alternatives/suggestion how to ensure that though.

Reply:
I agree it would be nice but it is not required. In most of the cases will be noop.